src/index.html

```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="header__logo" href="#Home">
+          <img class="logo"
+          src="images/log.png"
+          alt="Logo site">
+        </a>
+        <nav class="header__nav">
+          <ul class="nav_list">
+            <li class="menu_item"><a class="menu_link is-active" href="#">Apple</a></li>
+            <li class="menu_item"><a href="#" class="menu_link">Samsung</a> </li>
```

Comment:
```suggestion
            <li class="menu_item">
              <a class="menu_link is-active" href="#apple">Apple</a>
            </li>
            
            <li class="menu_item">
              <a href="#samsung" class="menu_link">Samsung</a>
            </li>
```

Fix formatting, update links and add blank line between multiple sibling blocks of html